slang-antlr/src/main/antlr4/com/sonarsource/slang/parser/SLang.g4

```diff
@@ -55,15 +55,34 @@ variableDeclaratorId
 
 methodBody
   : block 
-  | SEMICOLON 
+  | SEMICOLON
   ;
 
 block
-  :  LCURLY (statementOrExpression semi)* (statementOrExpression semi?)? RCURLY 
+  :  LCURLY (statement semi)* (statement semi?)? RCURLY
```

Comment:
Should we keep `semi` and `semi?` here since now we enforce `;` on top level satements?
I.e. should we enforce semicolon here too?

Reply:
Yes, I think this is good.